metron-deployment/roles/ambari_config/vars/single_node_vm.yml

```diff
@@ -80,10 +88,32 @@ configurations:
   - kafka-broker:
       log.dirs: '{{ kafka_log_dirs }}'
       delete.topic.enable: "true"
+  - metron-env:
+      parsers: "bro,snort"
+  - elastic-site:
+      index_number_of_shards: 1
+      index_number_of_replicas: 0
+      zen_discovery_ping_unicast_hosts: "{{ groups.search | join(',') }}"
+      gateway_recover_after_data_nodes: 1
+      network_host: "_lo_,_eth0_,_eth1_"
+      masters_also_are_datanodes: "1"
```

Comment:
Scratch the scratch. It needs to be "1" or ""true"" for now. It's the same ES problem we talked about above. If you want me to look at move the "translate True to ""true"" " test to params, I can do that, but it's going to need to exist somewhere.